server/constants/vars.js

```diff
@@ -9,6 +9,7 @@ export const INCLUSION_EXCLUSION_CRITERIA_FORM =
 export const SOCIODEMOGRAPHICS_FORM = 'form_sociodemographics'
 export const TOTAL_LABEL = 'Total'
 export const SITE = 'Site'
+export const INITIAL_ROLE = 'member'
```

Comment:
should this be defined in the UserModel?

Reply:
[88c0683](https://github.com/AMP-SCZ/dpdash/pull/293/commits/88c0683e88c4bba873baf2a78d32f796c0c889a5)